client/src/components/Projects/MultiProjectToolbarMenu.jsx

```diff
@@ -170,8 +171,10 @@ const MultiProjectToolbarMenu = ({
             }`}
             onClick={handlePrintPdf}
             disabled={checkedProjectIds.length !== 1}
+            aria-label="Print PDF"
           >
             <MdPrint
+              aria-hidden="true"
               className={
```

Comment:
add `aria-disabled` so that screen reader will read out if a button is disabled. set it to `checkedProjectIds.length !== 1`